lib/scala/launcher/src/main/scala/org/enso/launcher/Main.scala

```diff
@@ -0,0 +1,50 @@
+package org.enso.launcher
+
+import java.nio.file.Path
+
+import cats.implicits._
+import com.monovore.decline._
+
+object MainCommandBuilder {
+  private val JSON_OPTION = "json"
+
+  private def newCommand: Command[Unit] =
+    Command("new", "Create a new Enso project.") {
+      val nameOpt = Opts.argument[String]("name")
+      val pathOpt = Opts.argument[Path]("path").orNone
+
+      (nameOpt, pathOpt).mapN { (name, path) =>
+        Launcher.newProject(name, path)
+      }
+    }
+
+  private def versionCommand: Command[Unit] =
```

Comment:
`./enso --version` should also work here

Reply:
So should `enso version` and `enso --version` both work?

Reply:
Yes